src/messages/MPGStats.h

```diff
@@ -50,6 +53,9 @@ class MPGStats : public PaxosServiceMessage {
     ::encode(pg_stat, payload);
     ::encode(epoch, payload);
     ::encode(had_map_for, payload);
+    
+    //FIXME: conditional encoding depending on features?
```

Comment:
no need; old code will just ignore the tail of the message.  the feature-based encoding is only needed if something in teh existing encoding changes format.